core/workspace_comment/workspace_comment_render_svg.js

```diff
@@ -30,4 +30,134 @@ goog.require('Blockly.WorkspaceCommentSvg');
 
 Blockly.WorkspaceCommentSvg.prototype.render = function() {
   console.log('Rendering a comment!');
+
+  this.renderDraw_();
+};
+
+/**
+ * Draw the path of the block.
+ * Move the fields to the correct locations.
+ *     containing position information.
+ * @private
+ */
+Blockly.WorkspaceCommentSvg.prototype.renderDraw_ = function() {
```

Comment:
No need to have `renderDraw_` be different from `render`, since there's no compute step.